easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2024.06-foss-2023b.eb

```diff
@@ -0,0 +1,3451 @@
+    }),
+    ('graphlayouts', '1.1.1', {
+        'checksums': ['7bc2459a02b1339ac01184a76687a3e50de5680f4699b5966a3f2e6a882f3801'],
+    }),
+    ('tweenr', '2.0.3', {
+        'checksums': ['efabe512a45d653787ba40f87f3e23add4037f88573a102fa9ac7a5ff43c8cbe'],
+    }),
+    ('ggforce', '0.4.2', {
+        'checksums': ['c145b0e6ed6847d409ed2fe103b81234855bc0661cde2bfb4410fb23680e51a8'],
+    }),
+    ('tidygraph', '1.3.1', {
+        'checksums': ['aac1d4bb9396081bbeecbde11a3cd1a26a56bd6b1f608a628b359cb37c18ac1a'],
+    }),
+    ('ggraph', '2.2.1', {
+        'checksums': ['4405f8a907ad8fee68b5d4991f0bc8f35d6c0facbb7467c2ce425d3ec8b23af1'],
+    }),
+    ('qgraph', '1.9.8', {
+        'checksums': ['14a81d64f37614a05445408babbb2da5bc53886def8b0c2e4101b06e8b4c01d4'],
+    }),
+    ('HWxtest', '1.1.9', {
+        'patches': ['HWxtest-1.1.9_add-fcommon.patch'],
+        'checksums': [
+            {'HWxtest_1.1.9.tar.gz': 'a37309bed4a99212ca104561239d834088217e6c5e5e136ff022544c706f25e6'},
+            {'HWxtest-1.1.9_add-fcommon.patch': '4ce08c35035dbcc4edf092cdb405ae32c21c05b3786c15c0aa4bfe13bd81f451'},
+        ],
+    }),
+    ('diveRsity', '1.9.90', {
+        'checksums': ['b8f49cdbfbd82805206ad293fcb2dad65b962fb5523059a3e3aecaedf5c0ee86'],
+    }),
+    ('doSNOW', '1.0.20', {
+        'checksums': ['917cabed166aa2d1ec291691c17e1e3d344e858543e1682e3a442cc0c504bbb8'],
+    }),
+    ('geepack', '1.3.10', {
+        'checksums': ['4a38923b3697fd5e6488075224b63ef10195a29223e73a40e969f7db63215f42'],
+    }),
+    ('biom', '0.3.12', {
+        'checksums': ['4ad17f7811c7346dc4923bd6596a007c177eebb1944a9f46e5674afcc5fdd5a1'],
+    }),
+    ('pim', '2.0.2', {
+        'checksums': ['1195dbdbd67348dfef4b6fc34fcec643da685ebe58d34bbe049ab121aca9944f'],
+    }),
+    ('minpack.lm', '1.2-4', {
+        'checksums': ['e30fa4fe353cf00d266839d3c5db83ec9548a660f31d447ad9a69f556d56e731'],
+    }),
+    ('rootSolve', '1.8.2.4', {
+        'checksums': ['e16a317ea494192e0a5668a18f7eb99675f8edf3b3095861d213bc2590ad385d'],
+    }),
+    ('FME', '1.3.6.3', {
+        'checksums': ['83c4c28ad4f9197610be40fb66f1025f438a46e4085d64b736e83a0ab71e36a1'],
+    }),
+    ('bmp', '0.3', {
+        'checksums': ['bdf790249b932e80bc3a188a288fef079d218856cf64ffb88428d915423ea649'],
+    }),
+    ('tiff', '0.1-12', {
+        'checksums': ['df10ce719f92597572763182f7cb03686b8d7fb9123d036a4daf5b10738e815c'],
+    }),
+    ('readbitmap', '0.1.5', {
+        'checksums': ['737d7d585eb33de2c200da64d16781e3c9522400fe2af352e1460c6a402a0291'],
+    }),
+    ('imager', '1.0.2', {
+        'checksums': ['7c849086cb17d6c5aefc106217363e14afbcda2a9e0120687d40805b5e1c566a'],
+    }),
+    ('signal', '1.8-0', {
+        'checksums': ('0a604949bae91410a150a22cfa02d954f5b83166cc7a73e5409554d00e0417a7',   # older source
+                      '23ff303856ff09734e8ef280cec94556a15cdc702727ae5d02f8cfa4d8f473b7'),  # newer source
```

Comment:
```suggestion
        'checksums': [
            ('89cba854167a2b051a58cf3b73ccbf74eeb47c890ac39720611cd41f86b94684',
             '0a604949bae91410a150a22cfa02d954f5b83166cc7a73e5409554d00e0417a7'),
        ],
```

Reply:
Or this, yes. To me it looks better on 2 lines for the single list element though. I also took the liberty to improve the error message seen here: https://github.com/moravveji/easybuild-easyconfigs/pull/2

Reply:
I see the many green lights again. So, thanks @Flamefire  for the nice fix.
All the necessary adjustments after the recent merges are included here.
So, it this PR ready for testing?

Reply:
Test report coming up. I don't expect any issues. Just takes a while to install 1115 extensions ;-)

Reply:
I have seen surprises at test step ... so, I stay tuned. On our machines it took roughly 6 hours to build (including dependencies).